jwst_mast_query/jwst_download.py

```diff
@@ -96,9 +96,10 @@ def download_product(self, url, outfilename, clobber=False):
             print('ERROR: outfilename is empty string!')
             return(4,'ERRORempty')
 
-        if os.path.exists(outfilename) and not clobber:
-            print(f'WARNING: {outfilename} exists and clobber=False, thus skipping re-downloading it!')
-            return(2,'exists')
+#        if os.path.exists(outfilename):
```

Comment:
Did you meant to leave this commented out?

Reply:
Yes! this check prevented the code to overwrite corrupted files